lxd/daemon.go

```diff
@@ -1207,6 +1212,29 @@ func (d *Daemon) init() error {
 		return err
 	}
 
+	// Setup and load the servers UUID file.
+	// Use os.VarDir to allow setting up the uuid file also in the test suite.
+	var serverUUID string
+	uuidPath := filepath.Join(d.os.VarDir, "server.uuid")
+	if !shared.PathExists(uuidPath) {
+		serverUUID = uuid.New().String()
+		err := os.WriteFile(uuidPath, []byte(serverUUID), 0644)
```

Comment:
Does it need to be globally readable?

Reply:
Not that I am aware of. I got inspired by the permissions of `server.crt` in the same directory. Do you know why those are globally readable?

Reply:
im not sure. I think I'd be tempted to lock it down to root readable only and the relax it later if needed.

Reply:
Fine for me, fixed.